src/index.ts

```diff
@@ -1,4 +1,4 @@
-import { SupabaseStorageClient } from './SupabaseStorageClient'
+import { StorageClient } from './StorageClient'
 
-export { SupabaseStorageClient }
+export { StorageClient as StorageClient, StorageClient as SupabaseStorageClient }
```

Comment:
Optional: `StorageClient` instead of `StorageClient as StorageClient`, re-export instead of importing then exporting.

Reply:
Resolved in [this commit](https://github.com/supabase/storage-js/pull/48/commits/56ebc1849cb0dc83ea68bfb189cfe95d3d900df9)